etna/lib/etna/clients/magma/workflows/update_attributes_from_csv_workflow.rb

```diff
@@ -53,7 +53,11 @@ def models
 
       class RowBase
         def stripped_value(attribute_value)
-          attribute_value ? attribute_value.strip : attribute_value
+          attribute_value = attribute_value&.strip
+          if attribute_value && @workflow.json_values
+            attribute_value = JSON.parse(attribute_value)
+          end
+          attribute_value
```

Comment:
Maybe here (possibly elsewhere?), I think you'll need to detect a `file` or `file_collection` attribute from the model template and set the revision value to actually be:

```
{ path: <the input metis path from the file,
 original_filename: <the filename extracted from the metis path>
}
```
And an array of those objects for `file_collection`.

Reply:
right, so this will be handled by the fact that when generating the file linking csv, columns will all be JSON string values, and encoded in that way.

Reply:
Ah, I see. Nice!